.github/workflows/book.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Setup mdbook
         run: |
           cargo install --version 0.4.40 mdbook
-          cargo install mdbook-mermaid mdbook-template
+          cargo install mdbook-mermaid mdbook-template mdbook-alerts
```

Comment:
TIL about `mdbook-alerts`. Neat that people are pulling in GH flavored markdown.